### DISCUSSION TITLE
create a custom report and send to an email list

### DISCUSSION THREAD

**Discussion by chuyengiason** [mitmproxy/mitmproxy#5765] [Ideas]
Hi. 
I would like to know how do you store the capture data. So that I can create a job to read them, create a costumed report and send to a configured email list.

**Comment by mhils** [comment]
https://github.com/mitmproxy/mitmproxy/blob/main/examples/addons/io-read-saved-flows.py

**Reply by chuyengiason** [reply]
hi 
I can see the dump file but don't understand the structure of it.
Do u have document about the fields that mitmproxy can read and save ?
